### PR TITLE
Fix up product name & remove the c++ copied dir

### PR DESCRIPTION
**Pull request by perry-ca**
The product name for the Open Enterprise product  was incorrect and would lead to confusion with the actual Open XL product.

Eliminate the copy of the header file from the c++ compiler.  The problem has been fixed.  Keeping the copy will lead to problems.

### CODE REVIEW

**Review by gabylb — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ ZOSLIB is supported on the following hardware:
 * CMake 3.24.2-zos+
 * GNU Make 4.3+
 * IBM XL C/C++ V2.3.1 for z/OS V2.3 web deliverable (xlclang/xlcang++) or
-  IBM Open XL C/C++ 2.0 (clang/clang++)
+  IBM C/C++ for Open Enterprise Languages on z/OS 2.0.0 (clang/clang++)
```
Comment (gabylb) [inline README.md:55]: Shouldn't both be listed, and with XL stating (ibm-clang/ibm-clang++)?

File: zoslib.gyp [hunk 2]
```diff
@@ -32,7 +32,7 @@
           'include_dirs': ['include'],
         }, {
           'cflags': ['-fzos-le-char-mode=ascii', '-fgnu-keywords', '-fno-short-enums', '-mzos-target=zosv2r4'],
-          'include_dirs': ['include', 'include/c++/v1'],
+          'include_dirs': ['include'],
```
Comment (gabylb) [inline zoslib.gyp:35]: Can remove the include_dirs on line 32 and move this one to before sources below.

Comment (gabylb) [reply zoslib.gyp:35]: CMakeLists.txt also has `include_directories(BEFORE include include/c++/v1)` that should be removed.

Comment (perry-ca) [reply zoslib.gyp]: Thanks.  made the changes.

File: README.md (hunk 1 above)
Comment (perry-ca) [reply README.md]: I added Open XL v2.1 too.  One thing I did notice is that the -m64 option is not being passed to the link step.  That's a problem in the CMakefile.txt.  We'll need to fix that.  For now people can just use ibm-clang64 as the command.

**Comment by perry-ca** [comment]
@gabylb Are there any other branches that I should update.  It seems zopen2 is the defacto main.  Should I update main too?

**Comment by gabylb** [comment]
> @gabylb Are there any other branches that I should update. It seems zopen2 is the defacto main. Should I update main too?

I'm not aware of any. @IgorTodorovskiIBM ?

**Comment by IgorTodorovskiIBM** [comment]
> > @gabylb Are there any other branches that I should update. It seems zopen2 is the defacto main. Should I update main too?
> 
> I'm not aware of any. @IgorTodorovskiIBM ?

I would like to merge zopen2 once we stabilize it. Gaby, are you using the main branch for node?

**Pull request merged**